### DISCUSSION TITLE
Not Loading

### DISCUSSION THREAD

**Discussion by zamarax** [ramensoftware/windhawk#9] [Q&A]
Hi All, I have an issue on one of my laptops where WindHawk just loads with a blank screen thus I can't browse and install MOD's. See screenshot below.

![image](https://user-images.githubusercontent.com/25627568/179065757-20e9d5ce-34cc-49e5-8ade-bc64b7877a2f.png)

**Comment by m417z** [comment]
Please try the following:
* Press Ctrl+Alt+Shift+P.
* Type `>Toggle Developer Tools` and press enter.
* Open the Console tab and see whether there are any errors.

**Reply by zamarax** [reply]
Hi, I tried that but nothing happens - I noticed on a working machine that brings up web developer tools, are there any specific firewall policies or security settings that need to be set to allow Windhawk to run?

**Reply by m417z** [reply]
No settings should be needed. If you have security software like an antivirus, you might want to temporarily turn it off and check whether Windhawk works, perhaps it's being blocked for some reason.

Also, you can try running "C:\Program Files\Windhawk\UI\VSCodium.exe" directly and see whether it works. Windhawk uses [VSCodium](https://vscodium.com/) for the UI, and perhaps there's a problem with it.

**Comment by zamarax** [comment]
Seems to load in VSCode per screenshot, I'm going to whitelist VSCodium and try again.

![image](https://user-images.githubusercontent.com/25627568/179234867-f5e70dda-883e-4740-94d6-861876c7af08.png)

**Comment by m417z** [comment]
There's a similar issue reported here: https://github.com/ramensoftware/windhawk/issues/26

I'm trying to gather some information. If you still can reproduce the issue, please see [my comment here](https://github.com/ramensoftware/windhawk/issues/26#issuecomment-1365038267) and try to run Windhawk with logs enabled, hopefully they'll show the reason for the problem.
